c3.php

```diff
@@ -8,420 +8,463 @@
  * @author tiger
  */
 
-// $_SERVER['HTTP_X_CODECEPTION_CODECOVERAGE_DEBUG'] = 1;
+namespace Codeception;
 
 use SebastianBergmann\CodeCoverage\CodeCoverage;
 use SebastianBergmann\CodeCoverage\Driver\Driver;
 use SebastianBergmann\CodeCoverage\Driver\Selector;
 use SebastianBergmann\CodeCoverage\Filter as CodeCoverageFilter;
+use Codeception\Configuration;
+use PHP_CodeCoverage;
 
-if (isset($_COOKIE['CODECEPTION_CODECOVERAGE'])) {
-    $cookie = json_decode($_COOKIE['CODECEPTION_CODECOVERAGE'], true);
+// $_SERVER['HTTP_X_CODECEPTION_CODECOVERAGE_DEBUG'] = 1;
```

Comment:
I guess this one was there as a configuration option. How to configure it now if the file isn't yours?

Reply:
This (commented) code was only moved from above the use-statements to below. Functionality is unchanged.